scripts/fetch-docs.js

```diff
@@ -93,9 +98,20 @@ const fetchPages = async () => {
     // NOTE: this should be returned in the JSON payload
     const type = input.find(({ ids }) => ids.includes(doc.docId)).type;
     const frontmatter = getFrontmatter(type, doc);
+    const turndownService = new TurndownService({
+      headingStyle: 'atx',
+    });
+    turndownService.addRule('codeBlocks', {
+      filter: ['pre'],
+      replacement: function (content) {
+        // return '```' + '\n' + content + '```' + '\n';
```

Comment:
Can we remove this line?

Reply:
ah yep. will remove the commented out return now